### ISSUE TITLE
AttributeError: 'OpenProject' object has no attribute 'get_user_services'

### ISSUE THREAD

**Issue by jeremybep** [Flying-Free/pyopenproject#32]
Hi,

I've just copy/paste the code bloc of readme and i'v got error : 

`AttributeError: 'OpenProject' object has no attribute 'get_user_services'`

Some module changed ? is really hard to follow with test case, have you a documentation ?
or simple exemple for beginning ?

regards,

**Comment by steveblamey** [comment]
I think it should be `get_user_service`, without an 's' at he end.
